### PR TITLE
Don't show non-rep'ed artists when they set to no display

### PR DESCRIPTION
**Pull request by gnilekaw**
Fixes https://github.com/artsy/force/issues/316

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: apps/partner/templates/_artists_grid.jade [hunk 1]
```diff
@@ -2,7 +2,7 @@
   each group in groups
     section.artists-grid-section
       h2.avant-garde-header-center.artists-group-label= group.label
-      .partner-artists-conatiner
+      .partner-artists-container
```
Comment (mzikherman) [inline apps/partner/templates/_artists_grid.jade]: thnak you

File: apps/partner2/components/artists_grid/view.coffee [hunk 2]
```diff
@@ -20,7 +20,7 @@ module.exports = class PartnerArtistsGridView extends Backbone.View
         # Display represented artists or non- ones who have published artworks
         partnerArtists.filter (pa) ->
           pa.get('represented_by') or
-          pa.get('published_artworks_count') > 0
+          (pa.get('published_artworks_count') > 0 and pa.get('display_on_partner_profile') is true)
```
Comment (mzikherman) [inline apps/partner2/components/artists_grid/view.coffee]: http://www.trueistrue.com/

#minor can just be `pa.get('display_on_partner_profile')`, no need to compare it to true

**Comment by mzikherman** [comment]
Nice digging!

File: apps/partner/templates/_artists_grid.jade (hunk 1 above)
Comment (gnilekaw) [inline apps/partner/templates/_artists_grid.jade]: 🙃

File: apps/partner2/components/artists_grid/view.coffee (hunk 2 above)
Comment (gnilekaw) [inline apps/partner2/components/artists_grid/view.coffee]: Oh, just changed this, but changes already merged!

**Pull request merged**